service/agentconf_test.go

```diff
@@ -259,17 +266,18 @@ func (s *agentConfSuite) TestWriteServiceCopyStartAllAgents(c *gc.C) {
 	s.assertToolsCopySymlink(c, "xenial")
 }
 
-func (s *agentConfSuite) TestPreRebootWriteServiceCopyStartAllAgents(c *gc.C) {
+func (s *agentConfSuite) TestWriteServicePreRebootCopyStartAllAgents(c *gc.C) {
```

Comment:
This is not a new method but `TestWriteServicePreRebootCopyStartAllAgents` doesn't quite give away the purpose of the test. This could be a symptom of the test doing too much and perhaps should be broken down into a series a simpler tests.

Reply:
I broken down the tests into small pieces.